namespaces/live.cloud-platform.service.justice.gov.uk/parliamentary-questions-production/resources/service-account-github.tf

```diff
@@ -1,16 +1,15 @@
-module "serviceaccount" {
+module "serviceaccount-github" {
   source = "github.com/ministryofjustice/cloud-platform-terraform-serviceaccount?ref=1.1.0"
 
   namespace          = var.namespace
   kubernetes_cluster = var.kubernetes_cluster
 
-  github_repositories = [var.repo_name]
-
-  # This module is also used in the staging namespace, so we must change the
-  # secret names here in order to not overwrite the existing ones.
+  serviceaccount_name = "github-serviceaccount"
+  role_name = "github-serviceaccount"
+  rolebinding_name = "github-serviceaccount"
 
-  github_actions_secret_kube_cert      = "KUBE_PROD_CERT"
```

Comment:
I'm assuming we dont need these anymore if we are using a Github production environment to store the values